fakestorage/upload.go

```diff
@@ -345,6 +359,7 @@ func (s *Server) resumableUpload(bucketName string, w http.ResponseWriter, r *ht
 // it can't process a native "308 Permanent Redirect". The in-process response
 // then has a status of "200 OK", with a header "X-Http-Status-Code-Override"
 // set to "308".
+//nolint:funlen
```

Comment:
I just pushed a change on the main branch disabling funlen globally, can you remove this line? (sorry that our golangci-lint config is so annoying, I'm planning to eventually review it and simplify it - or just go with staticcheck + some very light checks in golangci-lint - mainly because gofmt won't exit with a non-0 status when it fails :/)

Reply:
Thanks @fsouza! Looks like you already got that one? https://github.com/fsouza/fake-gcs-server/commit/6ef4f624250ad255d3d435c725ce858c54b1a66c

Reply:
Yep, we're all good!